packages/app/src/runs/RunsError.spec.tsx

```diff
@@ -9,12 +9,12 @@ describe('<RunsError />', () => {
         return (
```

Comment:
Nit and unrelated, but can we rename this `.cy.tsx` instead? I'm not sure why it's using `.spec`, we usually reserve this for tests that run in Node.js.

Reply:
I think there's a couple more that do this in `app`, but we could make this change in another PR - I just noticed it here, since this file had modifications.

Reply:
Looks like someone just did this on develop 👍🏻